services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -105,6 +107,8 @@ public void configure(Binder binder)
             Jerseys.addResource(binder, ClientInfoResource.class);
             LifecycleModule.register(binder, QueryResource.class);
             LifecycleModule.register(binder, DruidBroker.class);
+            binder.bind(LookupReferencesManager.class).in(ManageLifecycle.class);
+            LifecycleModule.register(binder, LookupReferencesManager.class);
```

Comment:
is binder.bind(..) necessary given that you are doing LifecycleModule.register(..) ?